user-docs/C/guide-default-apps.xml

```diff
@@ -10,64 +10,171 @@
 <!ENTITY language "&EnglishAmerican;">
 <!ENTITY language "C">
 ]>
+
 <chapter id="guide-default-apps">
-	<title>Quick guide to default applications</title>
+	<title>Default Applications</title>
+
+    <para>Xubuntu comes with a collection of preinstalled applications curated by the Xubuntu team which are accessible from the &appmenu; <guibutton>Applications Menu</guibutton> (<keycap>Ctrl + Escape</keycap>) as well as the <application>Application Finder</application> (<keycap>Alt + F3</keycap> or <keycap>Super + R</keycap>).</para>
+
+    <section id="favorites">
+        <title>Favorites [icon placeholder]</title>
+
+        <para>By default, Xubuntu displays a number of commonly used applications in the <guilabel>Favorites</guilabel> category of the &appmenu; <guibutton>Applications Menu</guibutton>. These applications include the software manager, word processor, and help documentation, as well as the default web browser and file manager. To add or remove entries from the <guilabel>Favorites</guilabel> category, right-click on an application entry and select the <guilabel>Add to Favorites</guilabel> or <guilabel>Remove from Favorites</guilabel> entry in the context menu.</para>
+
+        <note><para>To change the default web browser, mail reader, file manager, terminal emulator and more, open the <application>Default Applications</application> or <application>Preferred Applications</application> app.</para></note>
```

Comment:
Are there actually two launchers? The docs are version-specific, so we don't have to support earlier strings.

Reply:
There is just one launcher.